src/control/cmd/agent/main.go

```diff
@@ -186,12 +181,29 @@ func agentMain(log *logging.LeveledLogger, opts *cliOptions) error {
 		return err
 	}
 
+	enabled := atm.NewBool(!(os.Getenv("DAOS_AGENT_DISABLE_CACHE") == "true"))
```

Comment:
nit - `(os.Getenv("DAOS_AGENT_DISABLE_CACHE") != "true")` is a little clearer to me

Reply:
Ok, I've changed it.